components/cards/dashboardDataFieldTableCard.tsx

```diff
@@ -47,6 +48,7 @@ export const DashboardDataFieldTableCard: FC<CardProps> = ({
 
   return (
     <Card
+      className={styles.card}
```

Comment:
remove styles.card

add border-radius-10 class in global scss
use className=border-radius-10